psydac/linalg/tests/test_block.py

```diff
@@ -590,7 +590,7 @@ def test_3D_block_serial_basic_operator( dtype, npts, p, P1, P2, P3 ):
     assert np.allclose(A4.blocks[1][0]._data, (M2)._data/5,  rtol=1e-14, atol=1e-14 )
     assert np.allclose(A4.blocks[1][1]._data, (M1)._data/5,  rtol=1e-14, atol=1e-14 )
 #===============================================================================
-@pytest.mark.parametrize( 'dtype', [complex] )
+@pytest.mark.parametrize( 'dtype', [float, complex] )
```

Comment:
Yes the `dtype` variable should be a string in the api level because it is an information for pyccel. So it should be the string `'real'` or `'complex'`. 

In the case you just point out, `dtype` is the format of data used to stock the values. So it should be the DataType `float` or `complex`.